### ISSUE TITLE
Jittery, Jumpy, Distracting movement.

### ISSUE THREAD

**Issue by jordwalke** [atom/autocomplete-plus#842]
I have autocomplete configured to the default mode where it remains in one location as you type.
The autocomplete window does not remain in its correct location as results are populated. I've made a gif to illustrate. The autocomplete is a very important part of an IDE experience, and is central to the impression that Atom leaves. Fixing this small issue would make Atom feel much more solid.

![infuriating](https://cloud.githubusercontent.com/assets/977348/25875345/cf034540-34cb-11e7-9874-f1f1280ed7b4.gif)

**Comment by leroix** [comment]
@jordwalke I'm unable to reproduce this. Is it still a problem for you? If so, could you provide some reproduction steps?

**Comment by jordwalke** [comment]
Yes, it is still a problem for me. Any time I simply use autocomplete (I happen to be using Merlin/ocaml).
